lightning-c-bindings/Cargo.toml

```diff
@@ -17,7 +17,8 @@ crate-type = ["staticlib"
 [dependencies]
 bitcoin = "0.26"
 secp256k1 = { version = "0.20.1", features = ["global-context-less-secure"] }
-lightning = { version = "0.0.12", path = "../lightning" }
+# Note that the following line is matched by genbindings to update the path
+lightning = { git = "https://github.com/rust-bitcoin/rust-lightning", rev = "8b4ea56966878140b11aa3e17b44fcf1e01d6413" }
```

Comment:
Does it matter what the target repo is at? I'm seeing:

```term
+ g++ -Wall -Wno-nullability-completeness -pthread -std=c++11 -Wall -g -pthread demo.cpp -Ltarget/debug/ -lldk -ldl
demo.cpp:511:46: error: no matching function for call to 'get_route'
                        LDK::CResult_RouteLightningErrorZ route = get_route(ChannelManager_get_our_node_id(&cm1), &graph_2_ref, ChannelManager_get_our_node_id(&cm2), &outbound_channels, LDKCVec_RouteHintZ {
                                                                  ^~~~~~~~~
./include/lightning.h:8976:40: note: candidate function not viable: requires 9 arguments, but 8 were provided
struct LDKCResult_RouteLightningErrorZ get_route(struct LDKPublicKey our_node_id, const struct LDKNetworkGraph *NONNULL_PTR network, struct LDKPublicKey payee, struct LDKInvoiceFeatures payee_features, struct LDKCVec_ChannelDetailsZ *first_hops, struct LDKCVec_RouteHintZ last_hops, uint64_t final_value_msat, ...
                                       ^
1 error generated.
```

Reply:
Ah, maybe since that is pointing to a branch that hasn't been merged?

Reply:
Oh, oops, i must've accidentally ran it against the wrong branch. Let me merge the "drop C bindings" PR upstream and then rebuild.

Reply:
I was actually running against your branch. When I run it against a branch that was merged, everything works.